docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -71,7 +72,6 @@ \subsection{Stakeholders}
 \begin{itemize}
     \item Students of McMaster University
     \item Single income households
-    \item Developers of project, Jason Nam, Allan Fang, Sawyer Tang, Ryan Yeh
```

Comment:
imo what he means is developers are stakeholders, just not allan, jason and ryan....